lib/nvme/nvme_pcie.c

```diff
@@ -133,8 +133,8 @@ struct nvme_pcie_qpair {
 	/* Completion queue */
 	struct spdk_nvme_cpl *cpl;
 
-	LIST_HEAD(, nvme_tracker) free_tr;
-	LIST_HEAD(, nvme_tracker) outstanding_tr;
+	TAILQ_HEAD(, nvme_tracker) free_tr;
```

Comment:
In retrospect, a separate PR would have made sense. We changed to TAILQ in order to avoid iterating through outstanding_tr. All we need to do is check TAILQ_LAST to get the oldest tr. LIST did not allow us to do this. The only exception is for an admin_queue with outstanding AERs. In this case, we iterate backwards to check the oldest non-AER request.